src/main/java/dk/cngroup/lentils/controller/ProgressController.java

```diff
@@ -114,7 +123,7 @@ public String viewHintsForCypher(final @PathVariable("cypherId") Long cypherId,
         return HINT_LIST;
     }
 
-    @PostMapping("/takeHintOfCypher")
+    @PostMapping(value = "/takeHintOfCypher")
```

Comment:
"value =" bych tady asi vynechal, pokud to neni ve vsech kontrolerech takhle.

Reply:
odstranim u vsech